modules/ctl/src/main/scala/low/io.scala

```diff
@@ -0,0 +1,24 @@
+package gem.ctl
+package low
+
+import scalaz._
+import scalaz.effect._
+
+/** Low-level constructors for `CtlIO` operations related to system I/O. */
+object io {
+
+  case class Output(exitCode: Int, lines: List[String])
+
+  def exec(cmd: String \/ List[String], f: String => IO[Unit] = (_ => IO.ioUnit)): IO[Output] =
+    IO {
+      import scala.sys.process._
+      import collection.mutable.ListBuffer
+      val b = ListBuffer[String]()
+      val x = cmd.fold(_.cat, _.cat).run(ProcessLogger { s =>
```

Comment:
It's on the syntax from `scala.sys.process._` … it turns a string or list of strings into a `ProcessBuilder`. No idea where the name came from.